front/app/modules/commercial/representativeness/admin/containers/Dashboard/ChartCards.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+
+// hooks
+import useUserCustomFields from 'modules/commercial/user_custom_fields/hooks/useUserCustomFields';
+
+// components
+import ChartCard from '../../components/ChartCard';
+import EmptyCard from '../../components/ChartCard/EmptyCard';
+
+// utils
+import { isNilOrError } from 'utils/helperUtils';
+
+// typings
+import { IUserCustomFieldData } from 'modules/commercial/user_custom_fields/services/userCustomFields';
+
+// fake data
+import fakeData from './fakeData';
+
+// TODO remove this later, generalize for all select fields
```

Comment:
This is all placeholder stuff until we actually hook this up with the real backend. All of this is behind a feature flag so nobody will see this. In the future, we will render a `ChartCard` for each supported `customField`